src/main/java/com/conveyal/datatools/manager/models/transform/TypedFeedTransformation.java

```diff
@@ -0,0 +1,44 @@
+package com.conveyal.datatools.manager.models.transform;
+
+import com.conveyal.datatools.common.status.MonitorableJob;
+
+/**
+ * This abstract class is the base for arbitrary feed transformations.
+ */
+public abstract class TypedFeedTransformation<Target extends FeedTransformTarget> extends FeedTransformation {
```

Comment:
I'm pretty sure this whole class can be combined into the FeedTransformation class. See proposed PR: https://github.com/ibi-group/datatools-server/pull/379